src/main/resources/api.yaml

```diff
@@ -1136,6 +1188,42 @@ components:
           type: string
         otherProperties:
           type: string
+    CommentDto:
+      type: object
+      required:
+        - commentId
+        - ownerId
+        - rating
+        - title
+        - description
+      properties:
+        commentId:
+          type: string
+          format: uuid
+        ownerId:
+          type: string
+        rating:
+          type: integer
+          format: int16
+        title:
+          type: string
+        description:
+          type: string
+    CommentCreationDto:
+      type: object
+      required:
+        - ownerId
+        - rating
+      properties:
+        ownerId:
+          type: string
+        rating:
+          type: integer
+          format: int16
```

Comment:
Nitpick: at domain and persistance level this value has `short` type, maybe there is something like that in OpenAPI?